documentation/cloud-manager-guide/secure-data-connectors/add-data-connector.md

```diff
@@ -5,25 +5,37 @@ description: Add a data connector
 ---
 # Add a Data Connector
 
-This guide outlines the steps to add a new secure data connector in Environment Operations Center.
+The process to create a new secure data connector and configure it with data source requires the following high-level steps:
 
-[!info] The steps outlined in this guide assume you have already configured the OPA client to establish the WebSocket tunnel between your on-prem environment and Env Ops Center. For details on how to configure the OPA client, see the [configure an OPA client](...) guide. **(update link when available)**
+- The data connector must be created in Environment Operations Center.
+- A connection must be established between the on-prem or cloud network and the Env Ops Center data connector.
```

Comment:
Some guides use "on premises" and then this guide uses "on-prem", shortened and with a hyphen. Would like to see consistency here. Perhaps "on-premises" for the first mention and then "on-prem" for secondary/additional mentions? Either way, should also use a hyphen.